adapter-rest-contract/src/main/resources/io/github/fileanalysissuite/adapters/rest/contract/swagger.yaml

```diff
@@ -248,6 +247,18 @@ components:
         - fileContents
         - fileMetadata
 
+    RetryFileInfo:
+      type: object
+      properties:
+        fileId:
+          type: string
+        retryAfter:
+          type: string
+          pattern: '^P(?=\d+[YMWD])(\d+Y)?(\d+M)?(\d+W)?(\d+D)?(T(?=\d+[HMS])(\d+H)?(\d+M)?(\d+S)?)?$'
```

Comment:
You prefer to do this rather than just making it the number of seconds?

Reply:
Apparently, JSON schema will have a duration type in this format.

Reply:
So this is future-proofing?  Where are you getting this?

Reply:
Actually, json schema seems to support it already:
https://json-schema.org/understanding-json-schema/reference/string.html